src/pages/index.js

```diff
@@ -6,7 +6,7 @@ import Link from "@docusaurus/Link";
 import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
 import useBaseUrl from "@docusaurus/useBaseUrl";
 import styles from "./styles.module.css";
-import ActivityGraph from "./components/activity-graph/activity-graph";
+import ActivityGraph from "../components/activity-graph/activity-graph";
```

Comment:
Btw, you might need to move this import behind `<BrowserOnly>`, e.g. something like:

```js
    <BrowserOnly fallback={<div>Loading...</div>}>
      {() => {
        const ActivityGraph = await import("../components/activity-graph/activity-graph");
        return <ActivityGraph />;
      }}
    </BrowserOnly>
```

See https://docusaurus.io/docs/docusaurus-core/#browseronly-example-library

Reply:
Thanks for suggesting this mode, I had tried the basic BrowserOnly method but not that yet. The fix was this in case we run into it again:
```
<BrowserOnly fallback={<div>Loading...</div>}>
          {() => {
            const ActivityGraph = require("../components/activity-graph/activity-graph").default;
            return <ActivityGraph />;
          }}
</BrowserOnly>
```